website/docs/r/aws_integration.html.markdown

```diff
@@ -81,7 +81,7 @@ resource "signalfx_aws_integration" "aws_myteam" {
 * `regions` - (Optional) List of AWS regions that SignalFx should monitor.
 * `role_arn` - (Optional) Role ARN that you add to an existing AWS integration object. **Note**: Ensure you use the `arn` property of your role, not the id!
 * `services` - (Optional) List of AWS services that you want SignalFx to monitor. Each element is a string designating an AWS service. Conflicts with `namespace_sync_rule`. See the documentation for [Creating Integrations](https://developers.signalfx.com/integrations_reference.html#operation/Create%20Integration) for valida values.
-* `poll_rate` - (Optional) AWS poll rate (in seconds). Value between `60` and `300`.
+* `poll_rate` - (Optional) AWS poll rate (in seconds). Value between `60` and `600`.
```

Comment:
Since that is optional it would be nice to provide information what is the default value when not specified.